sources/packages/backend/libs/integrations/src/institution-integration/ier12-integration/ier12.processing.service.ts

```diff
@@ -251,6 +253,35 @@ export class IER12ProcessingService {
         disbursementAwards: this.getDisbursementAwards(
           disbursement.disbursementValues,
         ),
+        studentMaritalStatusCode:
+          workflowData.calculatedData.studentMaritalStatusCode,
+        studentAndSupportingUserContribution:
+          assessmentData.studentTotalFederalContribution +
+          assessmentData.studentTotalProvincialContribution +
+          (assessmentData.parentAssessedContribution ?? 0) +
+          (assessmentData.partnerAssessedContribution ?? 0),
+        parentExpectedContribution: assessmentData.parentAssessedContribution,
+        totalEligibleDependents:
+          workflowData.calculatedData.totalEligibleDependents,
+        familySize: workflowData.calculatedData.familySize,
+        parentalAssetContribution:
+          workflowData.calculatedData.parentalAssetContribution,
+        parentalContribution: workflowData.calculatedData.parentalContribution,
+        parentDiscretionaryIncome:
+          workflowData.calculatedData.parentDiscretionaryIncome,
+        studentLivingWithParents:
+          workflowData.studentData.livingWithParents === "yes",
```

Comment:
I think we have an enum for "yes" "no" , maybe it  instead of a direct string comparison

Reply:
Added a YesNoOptions enum specific to dynamic data coming from formio.